diff --git a/node_modules/react-native-edge-to-edge/android/build.gradle b/node_modules/react-native-edge-to-edge/android/build.gradle
index eab6d4e..70a9ce4 100644
--- a/node_modules/react-native-edge-to-edge/android/build.gradle
+++ b/node_modules/react-native-edge-to-edge/android/build.gradle
@@ -70,5 +70,5 @@ repositories {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation "com.facebook.react:react-native:+" // From node_modules
-    implementation "com.google.android.material:material:${safeExtGet("materialVersion", "1.12.0")}"
+    implementation "com.google.android.material:material:${safeExtGet("materialVersion", "1.10.0")}"
 }
